gap/SkeletalFinSetsForCAP.gi

```diff
@@ -285,11 +290,15 @@ end );
 ##
 AddIsLiftable( SkeletalFinSets,
   function ( f, g )
-    local im_g;
     
-    im_g := AsList( g );
+    f := AsList( f );
+    g := AsList( g );
+    
+    if 100 * Length( f ) < Length( g ) then
+        f := DuplicateFreeList( f );
```

Comment:
I think we should use `Set` here. Excerpt from the documentation:

DuplicateFreeList only uses the = comparison and will not sort the result. Therefore DuplicateFreeList can be used even if the elements of list do not lie in the same family. Otherwise, if list contains objects that can be compared with \< (31.11-1) then it is much more efficient to use Set (30.3-7) instead of DuplicateFreeList.

Reply:
Thanks for the hint :)